pulp_rpm/tests/functional/utils.py

```diff
@@ -206,14 +214,19 @@ def init_signed_repo_configuration():
     # create a new signing service
     utils_dir_path = os.path.dirname(os.path.realpath(__file__))
     signing_script_path = os.path.join(utils_dir_path, "sign-metadata.sh")
+
+    pub_key = pub_key.decode('utf-8')
+    pub_key_shell = pub_key.replace('\n', '')
```

Comment:
@bmbouter Here is my workaround for the issue I mentioned.

Unfortunately the signature is failing to verify - but this looks like it should be the last issue.

```
E           stderr: Error: Failed to download metadata for repo 'pulp3-source-fedora33.localhost.example.com_pulp_content_f12fe066-f3d5-45a2-a695-484367f05064_': repomd.xml GPG signature verification error: Bad GPG signature
E           Ignoring repositories: pulp3-source-fedora33.localhost.example.com_pulp_content_f12fe066-f3d5-45a2-a695-484367f05064_
E           Error: Unable to find a match: walrus
```

Reply:
Maybe the verification problem is *because* of the workaround, but I'm not sure how else to do it?

Reply:
Yeah don't strip out the newlines I think. What's the original issue you were trying to workaround?

Reply:
![Screenshot from 2021-06-09 16-18-50](https://user-images.githubusercontent.com/2751996/121423569-6aafe600-c93e-11eb-90d7-aee21c198a85.png)